src/utility/pathfinding.js

```diff
@@ -1,6 +1,13 @@
 import * as arrayUtils from './arrayUtils';
 
-// Start and end must be Hex type
+/** search
+ * @param {Hex} start ?
+ * @param {Hex} end ?
+ * @param {?} creatureSize ?
+ * @param {?} creatureId ?
+ * @param {?} grid ?
+ * @returns {?} ?
```

Comment:
Why so many ?

Reply:
please be my guest and actually supply the correct annotations! i don't understand the code base, let alone the individual code samples, to fix the documentation issues.

i am simply getting the linter to pass and then it can be iterated over with additional fixes.